<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelNoise</class>
 <widget class="QWidget" name="PanelNoise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:12pt; color:#0000ff;&quot;&gt;Parameters Panel of Noise&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>299</width>
     <height>162</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Orders:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinN">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
      <property name="value">
       <number>80</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="toolButtonN">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Window:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QComboBox" name="comboBoxWindow">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <item>
       <property name="text">
        <string>Rectangular</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tapered rectangular</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Triangular</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hanning</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hamming</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blackman</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Freq Low:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinFreqLow">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
      <property name="value">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="toolButtonFreqLow">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Freq High:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spinFreqHigh">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
      <property name="value">
       <number>15000</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QToolButton" name="toolButtonFreqHigh">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Amplitude:    </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinAmp">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2500000</number>
      </property>
      <property name="value">
       <number>1000000</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QToolButton" name="toolButtonAmp">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>uV</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
